src/afl-fuzz-run.c

```diff
@@ -802,6 +796,48 @@ void sync_fuzzers(afl_state_t *afl) {
 
     }
 
+    // now document the attempt to sync to this instance
+    sprintf(qd_synced_path, "%s/.synced/%s.last", afl->out_dir, sd_ent->d_name);
+    int id_fd2 =
+        open(qd_synced_path, O_RDWR | O_CREAT | O_TRUNC, DEFAULT_PERMISSION);
+    if (id_fd2 >= 0) close(id_fd2);
+
+    // It could be that the target syncing instance was restarted, check!
+    time_t      last_mtime = {};
+    char        id0[PATH_MAX];
+    glob_t      glob_result;
+    struct stat st;
+
+    if (stat(qd_synced_path, &st) == 0) { last_mtime = st.st_mtime; }
+
+    snprintf(id0, sizeof(id0), "%s/id:000000,*", qd_path);
+
+    if (glob(id0, 0, NULL, &glob_result) == 0 && glob_result.gl_pathc == 1 &&
```

Comment:
Need to globfree

Reply:
ah yes! had never worked with glob() before ...